tests/functional/common.js

```diff
@@ -66,11 +66,13 @@ export function reportRunCommandError(obj: Object, msg: ?string) {
   console.error(errorMessage);
 
   if (obj.stdout) {
+    obj.stdout = obj.stdout.replace('\\n', '\n');
     console.error('\nCommand stdout: ', obj.stdout);
```

Comment:
It's bad practice to re-assign object properties because something further down in the call chain could become confused. In other words, it could produce unwanted side effects. Instead, you can just replace the new lines before logging the argument: `console.error('\nCommand stdout: ', obj.stdout.replace('\\n', '\n'));`